### ISSUE TITLE
Row cycles have stopped working

### ISSUE THREAD

**Issue by dougmet** [datasciencecampus/sdg-indicators#281]
Mysterious new behaviour on GitHub Pages. I tried rolling back the whole site but it does no appear to be due to any change we've made. Front page currently looks like this:

![image](https://user-images.githubusercontent.com/5878305/29113574-4c6da8ee-7ce9-11e7-8d8a-69b8fb58c5dc.png)

and the indicators page have the same problem. Locally the site looks just fine, something appears to have changed on the GitHub pages side.

Every goal is going in its own row:
```
{% for goal in sdg_goals %}
        {% cycle 'add row' : '<div class="row no-gutters">', nil, nil, nil, nil, nil %}
```
instead of every six.

**Comment by nathaneastwood** [comment]
When you inspect the page, each element is actually the full width. I am wondering if something funny has happened with the css?

**Comment by dougmet** [comment]
Each goal has gone in its own row. So it's putting each one on a new line:

![image](https://user-images.githubusercontent.com/5878305/29115082-2ff3cb30-7cee-11e7-9be4-9e3a05ff5228.png)

**Comment by dougmet** [comment]
This is how it looks locally
![image](https://user-images.githubusercontent.com/5878305/29115135-5e67d970-7cee-11e7-8ac0-ad90f2e5f880.png)

**Comment by shaneporter** [comment]
There is also an issue on the Goal page, fixed by #282.

**Comment by dougmet** [comment]
Awesome! Thanks Shane, I should have checked my Jekyll versions. Will make a ticket to update the build etc. Thanks!
